matisse/src/main/java/com/zhihu/matisse/engine/ImageEngine.java

```diff
@@ -33,10 +33,10 @@
      * @param context     Context
      * @param resize      Desired size of the origin image
      * @param placeholder Placeholder drawable when image is not loaded yet
-     * @param imageView   ImageView widget
+     * @param iv   ImageView widget
      * @param uri         Uri of the loaded image
      */
-    void loadThumbnail(Context context, int resize, Drawable placeholder, ImageView imageView, Uri uri);
+    void loadThumbnail(Context context, int resize, Drawable placeholder, ImageView iv, Uri uri);
```

Comment:
My own favor.

First I set max length 100 and I modified some file ,however later I found `Matisse` has many lines over 100 while it's very hard to cut down these line's length.So I set max length 120 thus as you see some changes are bad.